switchboard/cli/src/config.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+use serde::Deserialize;
+use std::fs;
+use std::path::Path;
+
+#[derive(Debug, Deserialize)]
+pub struct Config {
+    pub api: Api,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Api {
+    pub url: String,
+}
+
+pub fn load_config(config_path: Option<&str>) -> Result<Config> {
+    let config_path = config_path.unwrap_or("~/.switchboard_config.toml");
```

Comment:
Can we use an XDG config path here instead, to avoid polluting home directories and being able to use this cross-platform?

See: https://docs.rs/xdg/latest/xdg/struct.BaseDirectories.html